captum/attr/_core/neuron/neuron_guided_backprop_deconvnet.py

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env python3
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+import torch.nn as nn
+from torch import Tensor
+
 from ..._utils.attribution import NeuronAttribution, GradientAttribution
 from ..._utils.gradient import construct_neuron_grad_fn
 from ..guided_backprop_deconvnet import GuidedBackprop, Deconvolution
 
 
 class NeuronDeconvolution(NeuronAttribution, GradientAttribution):
-    def __init__(self, model, layer, device_ids=None):
+    def __init__(
+        self, model: nn.Module, layer: nn.Module, device_ids: Optional[List] = None
```

Comment:
nit: We usually have been using the type as just Module and importing as "from torch.nn import Module", can you switch to that for consistency? Thanks!